simapp/ante.go

```diff
@@ -0,0 +1,39 @@
+package simapp
```

Comment:
why putting this in `simapp`?

Reply:
Can you suggest me a better place for this @blushi 
facing some import error `x/auth/ante` package.

Reply:
yeah moving it to `x/auth/ante` results in an import cycle  `x/auth/ante` ->  `x/fee_grant/ante` ->  `x/auth/ante` so either you'd need to move `x/fee_grant/ante` to `x/auth/ante` or `ante.go` to `x/fee_grant/ante`, not actually sure what's best.

Reply:
moving `ante.go` to `x/feegrant/ante` looks fine for me.